frame/collective/src/lib.rs

```diff
@@ -1114,36 +1124,56 @@ mod tests {
 	}
 
 	#[test]
-	fn proposal_weight_limit_works() {
+	fn proposal_weight_limit_works_on_approve() {
 		new_test_ext().execute_with(|| {
 			let proposal = Call::Collective(crate::Call::set_members(vec![1, 2, 3], None, MAX_MEMBERS));
```

Comment:
why not set the prime here?

Reply:
This is the proposal, not an actual call.

This was confusing to me in the test as well. I need the prime to ACTUALLY be set when we do the vote, whereas this is the proposal being voted on.

Reply:
Yeah, now that you mention it, I chose that call, maybeshould've picked something else :grimacing: